src/Scaffold/Console/model/model.stub

```diff
@@ -22,6 +24,39 @@ class {{studly_name}} extends Model
      */
     protected $fillable = [];
 
+    /**
+     * @var array Rules for fields
+     */
+    public $rules = [];
+
+    /**
+     * @var array Cast fields
+     */
+    protected $casts = [];
+
+    /**
+     * @var array Json fields
+     */
+    protected $jsonable = [];
+
+    /**
+     * @var array Append attributes
```

Comment:
`Attributes to be appended to the API representation of the model (ex. toArray())`